config/settings.js

```diff
@@ -27,11 +27,20 @@
  */
 
 window.spinnakerSettings = {
+  defaultProviders: ['aws', 'gce', 'azure', 'cf', 'kubernetes'],
+  feedbackUrl: process.env.FEEDBACK_URL,
   gateUrl: gateUrl,
-  bakeryDetailUrl: bakeryBaseUrl + '/api/v1/global/logs/{{context.status.id}}?html=true',
+  bakeryDetailUrl: bakeryBaseUrl + '/api/v1/global/logs/{{context.status.id}}?html=true'
+  authEndpoint: process.env.AUTH_ENDPOINT,
   pollSchedule: 30000,
-  defaultTimeZone: 'America/New_York', // see http://momentjs.com/timezone/docs/#/data-utilities/
+  defaultTimeZone: process.env.TIMEZONE || 'America/Los_Angeles', // see http://momentjs.com/timezone/docs/#/data-utilities/
```

Comment:
Will this actually pull values from `/etc/default/spinnaker`? I'm not sure how the individual processes startup. If not, I think new values should be added to the YAML config for things like timezone.